src/contracts/atlas/ExecutionEnvironment.sol

```diff
@@ -242,10 +242,9 @@ contract ExecutionEnvironment is Base {
 
         // Verify that the solver paid what they bid
         bool etherIsBidToken;
-        i = 0;
         uint256 balance;
 
-        for (; i < solverOp.bids.length;) {
+        for (uint i; i < solverOp.bids.length;) {
```

Comment:
Is this a gas optimization?

Reply:
No I dont think so, the uint i was defined for reuse outside of the loop but it was causing stack too deep errors by lingering around. So I just moved it there to contain it into scope of the loop

Reply:
ah, nice!